conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/data/tezos/TezosDataEndpoints.scala

```diff
@@ -1,16 +1,18 @@
-package tech.cryptonomic.conseil.routes.openapi
+package tech.cryptonomic.conseil.api.routes.platform.data.tezos
 
 import endpoints.algebra
-import tech.cryptonomic.conseil.tezos.{Filter, Tables}
-import tech.cryptonomic.conseil.tezos.Tables.BlocksRow
-import tech.cryptonomic.conseil.generic.chain.DataTypes._
-import tech.cryptonomic.conseil.tezos.ApiOperations.{AccountResult, BlockResult, OperationGroupResult}
+import tech.cryptonomic.conseil.api.routes.platform.data.tezos.TezosDataOperations._
+import tech.cryptonomic.conseil.api.routes.platform.data.tezos.TezosDataTypes.ApiQuery
+import tech.cryptonomic.conseil.api.routes.validation.Validation
+import tech.cryptonomic.conseil.common.generic.chain.DataTypes._
+import tech.cryptonomic.conseil.common.tezos.Tables.BlocksRow
+import tech.cryptonomic.conseil.common.tezos.Tables
 
 /** Trait containing endpoints definition */
-trait DataEndpoints
+trait TezosDataEndpoints
     extends algebra.JsonSchemaEntities
-    with DataJsonSchemas
-    with ApiFilterFromQueryString
+    with TezosDataJsonSchemas
+    with TezosApiFilterFromQueryString
     with Validation {
 
   /** Common path among endpoints */
```

Comment:
I think we have an issue here.

Differently from the metadata endpoints paths, where "platform" is a path segment, here we should have it fixed to the actual `"tezos"` value, because this routes will be combined with all those from any platform found in config, as I seem to understand from ConseilApi.scala L#90+

Reply:
at the moment(meaning before the big refactor) each query worked for each network found in config(TBH I've never tested with different platforms, but I suspect it would work/require small change). 
How I see it is that we should be able to use thee same routes(as the API is mostly generic) to use with all the platforms/networks and be overridden only with the metadata configuration.
There is also another thing here - those endpoints always use `platform` so renaming it to start with `Tezos` is misguiding.

Reply:
> There is also another thing here - those endpoints always use `platform` so renaming it to start with `Tezos` is misguiding.

Not sure I'm following you there, when asking for metadata I'm ok that the path would end on `platform` but then the user will swap that with `/tezos/networks` when digging deeper...

Same thing I expect for the data endpoints?

Like `/v2/data/tezos/mainnet/operations` ... 

The need here should be to swap the actual implementation based on the platform, because other blockchains might find data differently, AFAICT, so we build "custom routes" for each platform and mash them together under the same path ("/v2/data/") to get the specific implementations.
But each route implementation should provide a different segment for the platform. This is what I'm trying to tell.

Is there any error in what I'm saying?

Reply:
> Is there any error in what I'm saying?

no, thats more or less that I meant - create different segments for the platforms if we are going to get through with this redesign, and we should remove platform parameters from those endpoints.

Reply:
Keep in mind that `queryEndpoint` and everything what comes with that is "generic" so `platform` can be actually extracted and passed there, but only for this resource. Rest of the resources should be platform-specific, meaning `platform` segment should be removed from `path` and put as constant. 

I decided not to do that here, mostly because this changes the behavior of API. Currently, when platform is not supported, the lower layer will respond with such information and endpoint will convert this to 404. Once I'll hardcode "tezos" in a path name, the request with unsupported platform will be rejected (different error code or even exception). This requires dedicated error handling mechanism. I decided to limit the scope and leave it as it is. I can add comment, or even take care of such mechanism if you want to, but I don't know what changes it requires to be done. Need to investigate.

Reply:
Ok for me, I didn't get that you planned to enable different platforms in a later step.

This was just preliminary preparation, right?

Reply:
Yes